src/js/utils/VectorFactory.js

```diff
@@ -1,26 +1,48 @@
-export function geoJsonPolygonToSvg(geoJSON, boundingRect) {
-    let svg
-    if (geoJSON.substring(0, 9) === 'POLYGON((') {
-        svg = ''
-        const polygons = geoJSON.split("\),\(")
-        polygons.forEach(polygon => {
-            svg += 'M'
-            polygon = polygon.replace(/POLYGON/g, "")
-            polygon = polygon.replace(/\(/g, "")
-            polygon = polygon.replace(/\)/g, "")
-            var points = polygon.split(",")
-            points.forEach(point => {
-                if (svg.slice(-1) !== 'M'){
-                    svg += 'L'
-                }
-                svg += `${point.split(' ')[0] - boundingRect.x} ${point.split(' ')[1] - boundingRect.y}`
-            })
+/*
+ * This function transforms a well-known-text
+ * polygon into an SVG path.  When a boundingBox 
+ * is passed, then we translate every point based 
+ * on the x and y position of that bounding box.  
+ * Otherwise, we just add each point to the string 
+ * unaltered. 
+ */
+export function wktPolygonToSvg(geoJSON, boundingRect) {
```

Comment:
What does `wkt` mean or stand for?

Reply:
It is not so common a format wkt = Well Known Text, which is a geometric data encoding (like SVG).  It is the format that MariaDB uses internally for its geometric data.  Well actually MariaDB stores these as WKB (Well Known Binary).

Reply:
I see, ok. Thanks for filling me in.